packages/commonwealth/client/scripts/controllers/app/webWallets/cosmos_evm_metamask_web_wallet.ts

```diff
@@ -110,7 +104,7 @@ class CosmosEvmWebWalletController implements IWebWallet<string> {
       // (this needs to be called first, before other requests)
       const Web3 = (await import('web3')).default;
       this._web3 = new Web3((window as any).ethereum);
-      await this._web3.givenProvider.enable();
+      //await this._web3.givenProvider.enable();
```

Comment:
We should remove this line if its not necessary

Reply:
removed